enos/enos-dev-scenario-pr-replication.hcl

```diff
@@ -6,23 +6,130 @@ scenario "dev_pr_replication" {
+    scenario:
+
+      $ enos scenario outline dev_pr_replication
+
+    You can also create an HTML version that is suitable for viewing in web browsers:
+
+      $ enos scenario outline dev_pr_replication --format html > index.html
+      $ open index.html
+
+    # How to run this scenario
+
+    1. Install Enos (more info: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/)
+    
+      $ brew tap hashicorp/tap && brew update && brew install hashicorp/tap/enos
+
+    2. Authenticate to AWS with Doormat (more info: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/#authenticate-to-aws-with-doormat)
+    
+      $ doormat login && eval $(doormat aws -a <your_aws_account_name> export)
+
+    3. Set the following variables either as environment variables (`export ENOS_VAR_var_name=value`)
+    or by creating a 'enos-local.vars' file and setting it there (see enos.vars.hcl for examples):
+
+    Required variables
+      - aws_ssh_private_key_path (more info about AWS SSH keypairs: https://eng-handbook.hashicorp.services/internal-tools/enos/getting-started/#set-your-aws-key-pair-name-and-private-key)
+      - aws_ssh_keypair_name
+
+    Optional variables
+      - aws_region (set if different from the default value in enos-variables.hcl)
+      - dev_build_local_ui (set if different from the default value in enos-dev-variables.hcl)
+      - dev_consul_version (set if different from the default value in enos-dev-variables.hcl)
+      - distro_version_<distro> (set if different from the default version for your target
+        distro in enos-globals.hcl)
+      - vault_license_path (set if using an ENT edition of Vault)
+      - consul_license_path (set if using an ENT edition of Consul, according to var.backend_edition)
+    
+    4. Choose what type of Vault artifact you want to use, and set the appropriate variables.
+
+      a. 'artifact:local'
+      This will build a Vault .zip bundle from your local branch. Set the following variable:
+
+      vault_artifact_path = path/to/where/vault-should-be-built.zip
+
+      b. 'artifact:deb' or 'artifact:rpm'
+      This will download a Vault .deb or .rpm package from Artifactory with the version and
+      edition you specify. To do this, you will need to set your Artifactory credentials:
+
+      artifactory_username = your-user
+      artifactory_token = your-token
+
+      c. 'artifact:zip'
+      This will download a Vault .zip bundle from releases.hashicorp.com with the version and
+      edition you specify.
+
+      TODO: add required variable
+    
+    5. If you don't know yet what combination of matrix variants you want to use for your scenario, you 
+    can view all the possible combinations through the `list` command:
+
+      $ enos scenario list dev_single_cluster
+    
+    Once you know what filter you want to use to obtain your desired combination of matrix variants,
```

Comment:
Would be useful to mention  that you can reduce the list by providing partial filters?